src/relay/backend/build_module.cc

```diff
@@ -207,7 +210,7 @@ class RelayBuildModule : public runtime::ModuleNode {
     } else if (name == "optimize") {
       return PackedFunc([sptr_to_self, this](TVMArgs args, TVMRetValue* rv) {
         ICHECK_EQ(args.num_args, 2);
-        *rv = this->Optimize(args[0], args[1], this->params_);
```

Comment:
Why do we no longer need the parameters here?

Reply:
The code was using both local and member params simultaneousy, but the local was only ever bound to the member anyway. So I just got rid of the local.

Note there's some weirdness in this API as to what state is expected to come over between 'build' and 'optimize'.